syncd/RedisClient.cpp

```diff
@@ -48,21 +48,12 @@ std::string RedisClient::getRedisLanesKey(
      * NOTE: To support multiple switches LANES needs to be made per switch.
      *
      * return std::string(LANES) + ":" + sai_serialize_object_id(m_switch_vid);
-     *
-     * Only switch with index 0 and global context 0 will have key "LANES" for
-     * backward compatibility. We could convert that during runtime at first
-     * time.
      */
 
     auto index = VidManager::getSwitchIndex(switchVid);
 
     auto context = VidManager::getGlobalContext(switchVid);
 
-    if (index == 0 && context == 0)
-    {
-        return std::string(LANES);
-    }
-
```

Comment:
removing this will break compatibility with existing platforms and this solution already provides suppor for multiple asic

Reply:
why remove this will solve antything

Reply:
Hi Kamil - Thank you for taking a look. I have a Linecard with two ASICs. I think I need to change this piece of code because LANES mapping is needed for both syncd0 and syncd1. Without this change, the LANES mapping is created for only one chip and the port change on the second chip will fail because LANES mapping does not exist.

Reply:
if you have second syncd, then you need to have separate asic database for it, you cant use the same asic db for both, this is not supported, not only on lanes, but on many other components in entire syncd code

Reply:
i dont understand your problem, since you have 2 different namespaces for syncds there is no conflict there, and lanes mappings should be created correctly, if index and context is not zero, then just oid will be added, but for backward compatibility for index zero and context zero just LANES will be name

if you are saying that lanes are not created then the entry in DB is missong or what do you have in mind here?

Reply:
I think I understand your point now. the change in getRedisLanesKey might not be needed. I revert them in the latest commit.